package com.todo.todoapplication.domain.todo.exception

const val TODO_NOT_FOUND_MESSAGE = "조회한 TODO를 찾을 수 없습니다."
